worker/uniter/pebblewatcher.go

```diff
@@ -0,0 +1,266 @@
+// Copyright 2021 Canonical Ltd.
```

Comment:
Note that the first half of this file and the `runPoller` loop is existing code, but git didn't pick up the file rename as there were too many changes for the new notices-watcher stuff.